bin/api.wsgi

```diff
@@ -1,4 +1,9 @@
 # vim: filetype=python
+import sys
+import os.path
+
+repo_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../'))
+sys.path.append(repo_path)
```

Comment:
As far as I'm concerned, any kind of internal, *magic* manipulation of the PYTHONPATH is evil and should be avoided.